en/python_installation/instructions.md

```diff
@@ -9,9 +9,10 @@ Please install normal Python as follows, even when you have Anaconda installed o
 <!--sec data-title="Install Python: Windows" data-id="python_windows" data-collapse=true ces-->
 
 First check whether your computer is running a 32-bit version or a 64-bit version of Windows, on the "System type" line of the System Info page. To reach this page, try one of these methods:
-* Press the Windows key and Pause/Break key at the same time
+* Press the Windows key and start to type System Information, the menu should bring it up as best match within 3 letters, click on it or press enter to accept it 
```

Comment:
This should probably be three separate sentences:

```suggestion
* Press the Windows key and start to type System Information. The menu should bring it up as best match within 3 letters. Click on it or press enter to accept it.
```

Also, should "System Information", as the text to be typed in, be delimited in any way? E.g. put into quotes:
```suggestion
* Press the Windows key and start to type "System Information", the menu should bring it up as best match within 3 letters, click on it or press enter to accept it 
```
or with code / monospace markup:
```suggestion
* Press the Windows key and start to type `System Information`, the menu should bring it up as best match within 3 letters, click on it or press enter to accept it 
```